src/solvers/dgsem_tree/subcell_limiters.jl

```diff
@@ -41,62 +43,90 @@ The bounds are calculated using the low-order FV solution. The positivity limite
+            bound_keys = (bound_keys..., Symbol(v_string, "_min"),
+                          Symbol(v_string, "_max"))
+        end
+    end
+    for v in positivity_variables_cons
+        if !(v in local_minmax_variables_cons)
+            bound_keys = (bound_keys..., Symbol(string(v), "_min"))
+        end
+    end
 
     cache = create_cache(SubcellLimiterIDP, equations, basis, bound_keys)
 
-    SubcellLimiterIDP{typeof(positivity_correction_factor), typeof(cache)}(positivity,
-                                                                           positivity_variables_cons,
-                                                                           positivity_correction_factor,
-                                                                           cache)
+    SubcellLimiterIDP{typeof(positivity_correction_factor),
+                      typeof(cache)}(local_minmax, local_minmax_variables_cons,
+                                     positivity, positivity_variables_cons,
+                                     positivity_correction_factor, cache)
 end
 
 function Base.show(io::IO, limiter::SubcellLimiterIDP)
     @nospecialize limiter # reduce precompilation time
-    @unpack positivity = limiter
+    (; local_minmax, positivity) = limiter
 
     print(io, "SubcellLimiterIDP(")
-    if !(positivity)
+    if !(local_minmax || positivity)
         print(io, "No limiter selected => pure DG method")
     else
         print(io, "limiter=(")
+        local_minmax && print(io, "min/max limiting, ")
         positivity && print(io, "positivity")
         print(io, "), ")
     end
+    print(io, "Local bounds with FV solution")
     print(io, ")")
 end
 
 function Base.show(io::IO, ::MIME"text/plain", limiter::SubcellLimiterIDP)
     @nospecialize limiter # reduce precompilation time
-    @unpack positivity = limiter
+    (; local_minmax, positivity) = limiter
 
     if get(io, :compact, false)
         show(io, limiter)
     else
-        if !(positivity)
+        if !(local_minmax || positivity)
             setup = ["limiter" => "No limiter selected => pure DG method"]
         else
             setup = ["limiter" => ""]
+            if local_minmax
+                setup = [
+                    setup...,
+                    "" => "local maximum/minimum bounds for conservative variables $(limiter.local_minmax_variables_cons)",
+                ]
+            end
             if positivity
-                string = "positivity with conservative variables $(limiter.positivity_variables_cons)"
+                string = "positivity for conservative variables $(limiter.positivity_variables_cons)"
                 setup = [setup..., "" => string]
                 setup = [
                     setup...,
                     "" => "   positivity correction factor = $(limiter.positivity_correction_factor)",
                 ]
             end
+            setup = [
+                setup...,
+                "Local bounds" => "FV solution",
```

Comment:
If there is only one possible value here, why bother printing anything at all?

Reply:
There is another option to calculate the bounds. That's with the so-called bar states. I implemented that way in my branch, but it is not merged to main yet. Should I remove this information until there are two options?

Reply:
In general, yes, but if you know that this will be merged anyways within the next weeks, then I guess it's ok to leave it now